packages/fast-usdc/src/exos/liquidity-pool.js

```diff
@@ -178,7 +182,25 @@ export const prepareLiquidityPoolKit = (zone, zcf, USDC, tools) => {
           Object.assign(this.state, post);
           this.facets.external.publishPoolMetrics();
         },
-        // TODO method to repay failed `LOA.deposit()`
+        /**
+         * If something fails during advance, return funds to the pool.
+         *
+         * @param {ZCFSeat} borrowSeat
+         * @param {{ USDC: Amount<'nat'>}} amountKWR
```

Comment:
Is there any reason to make this a record?

Reply:
Only in as much `localTransfer`/`withdrawToSeat` and their ~counterparts `depositToSeat`, `withdrawToSeat` expect `amountKWR`.

I agree this could be dropped. If we did, I would do the same in `borrow`. Let me know if we should make this change.

Reply:
Yes, I think we should.

I'm OK with leaving it if it would cost you another ci round trip.